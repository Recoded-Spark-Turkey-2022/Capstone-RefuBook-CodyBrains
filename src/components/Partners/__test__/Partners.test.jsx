import React from 'react';
import { render } from '@testing-library/react';
import Partners from '../Partners';

jest.mock('swiper/react', () => ({
  Swiper: 'Swiper',
  SwiperSlide: 'SwiperSlide',
}));

jest.mock('swiper', () => ({
  Pagination: 'Pagination',
}));

jest.mock('../../../assets/companies/recoded.png', () => 'recoded');
jest.mock('../../../assets/companies/spark.png', () => 'spark');
jest.mock('../../../assets/companies/flatiron.png', () => 'flatiron');
jest.mock('../../../assets/companies/children.png', () => 'children');
jest.mock('../../../assets/companies/unhcr.png', () => 'unhcr');
jest.mock('../../../assets/companies/amnesty.png', () => 'amnesty');
jest.mock('../../../assets/companies/oxfam.png', () => 'oxfam');
jest.mock('../../../assets/companies/giz.png', () => 'giz');
jest.mock('../../../assets/companies/unicef.png', () => 'unicef');

describe('Partners', () => {
  it('renders correctly', () => {
    const { container } = render(<Partners />);
    expect(container).toMatchSnapshot();
  });
});
