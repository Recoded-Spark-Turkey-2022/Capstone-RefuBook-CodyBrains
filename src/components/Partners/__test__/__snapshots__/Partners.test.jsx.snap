// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partners renders correctly 1`] = `
<div>
  <div
    class="flex flex-col items-center justify-center w-full bg-refubookWhite px-8 py-4"
  >
    <h1
      class="text-3xl text-refubookBlack font-bold text-center"
    >
      Our Partners
    </h1>
    <swiper
      breakpoints="[object Object]"
      class="w-full "
      loop=""
      modules="Pagination"
      pagination="[object Object]"
      spacebetween="5"
    >
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="recoded"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="spark"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="flatiron"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="children"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="unhcr"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="amnesty"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="oxfam"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="giz"
        />
      </swiperslide>
      <swiperslide
        class="flex items-center justify-center "
      >
        <img
          alt="logo"
          class=" block w-40 h-40 object-contain  "
          src="unicef"
        />
      </swiperslide>
    </swiper>
  </div>
</div>
`;
